src/app/shared/alert/state/alert.service.ts

```diff
@@ -81,6 +81,20 @@ export class AlertService {
     this.matSnackBar.open(previousMessage + ' failed', 'Dismiss');
   }
 
+  /**
+   * Handles HTTP error response and show a detailed message only in the matSnackBar.
+   * Use when there's is no alert component which normally displays the detailed message.
+   *
+   * @param {HttpErrorResponse} error  The HttpErrorResponse return by the failed Http call
+   * @memberof AlertService
+   */
+  public detailedSnackBarError(error: HttpErrorResponse) {
+    this.clearEverything();
+    const detailedError = '[HTTP ' + error.status + '] ' + error.statusText + ': ' +
```

Comment:
I think I've commented on this elsewhere, and there may be a ticket for it already, but I don't think we should be surfacing HTTP status codes to the user, at least not by default -- maybe there is a detailed view where it gets shown. I would think the typical Dockstore user would know/understand http status codes...

But since we're doing it elsewhere, this is fine.

Reply:
I'm in the camp that I'd like to have them shown, if only to make it easier to debug bug reports. 
But that they can't be the only thing that is shown (i.e. a more descriptive message accompanying is  helpful)

Reply:
OK... The "debug bug reports" reminds me, one thing we did in a previous job was send all 4xx and 5xx errors in the browser to Google Analytics, then periodically try to cull the most common ones.